src/lines.ts

```diff
@@ -1,85 +1,107 @@
 import { start, line, wiggle, stationRange } from './paths';
 
+const enum GLStations {
```

Comment:
I'd be in favor of just writing this out as an object — TS development seems like it's moved away from using enums in recent years.

Reply:
Interesting, I wasn't aware of. I did some reading and couldn't find much (there's an [SO post](https://stackoverflow.com/a/66864086/11228445))